pettingzoo/utils/deprecated_module.py

```diff
@@ -47,7 +49,7 @@ def deprecated_handler(
                     if int(alt_version) > int(version):
                         return DeprecatedModule(name, version, alt_version)
                     else:
-                        raise ImportError(
+                        raise AttributeError(
```

Comment:
It might be worth adding a comment as to why it’s an attribute error not an import error, just for the future

Reply:
will do

Reply:
Done